pyiso/pjm.py

```diff
@@ -60,11 +81,55 @@ def fetch_edata_series(self, data_type, params=None):
 
         # parse html to df
         dfs = pd.read_html(response.content, header=0, index_col=0, parse_dates=True)
-        df = self.utcify_index(dfs[0])
+        df = self.utcify_index(dfs[0], tz_name='utc')
 
         # return df
         return df
 
+    def fetch_historical_load(self, year):
+        url = 'http://www.pjm.com/pub/operations/hist-meter-load/%s-hourly-loads.xls' % year
+        response = self.request(url)
+        dfs = pd.read_excel(StringIO(response.content), sheetname=None)
+
+        # Strip all subregions, except PJM RTO total
+        df = dfs['RTO']
+        drop_col = ['Unnamed: 0', 'Unnamed: 27', 'Unnamed: 28', 'Unnamed: 29', 'Unnamed: 30',
+                    'MAX', 'HOUR', 'DATE.1', 'Unnamed: 34', 'MIN', 'HOUR.1', 'DATE.2']
+        df.drop(drop_col, axis=1, inplace=True)
+
+        df = pd.melt(df, id_vars=['DATE', 'COMP'])
+
+        # Get datetime format
+        df['hour'] = df['variable'].str.strip('HE').astype(int) - 1
+        df['datetime_str'] = (pd.to_datetime(df['DATE']).astype(str) + ':' +
+                              df['hour'].astype(str).str.zfill(2))
+        df['timestamp'] = pd.to_datetime(df['datetime_str'], format='%Y-%m-%d:%H')
+
+        # todo handle DST transitions properly, this just returns Not a Time
+        f = lambda x: pytz.timezone(self.TZ_NAME).localize(x['timestamp'])
+        df['timestamp'] = df.apply(f, axis=1)
+
+        # pandas time series functions only work on index
+        df.index = df['timestamp']
+        df['timestamp'] = df.index.tz_convert('utc')
+
+        drop_col = ['datetime_str', 'DATE', 'hour', 'variable', 'COMP']
+        df.drop(drop_col, axis=1, inplace=True)
+
+        # add formatting
+        extras = {
+                'freq': self.FREQUENCY_CHOICES.hourly,
+                'market': self.MARKET_CHOICES.dam,
+                'ba_name': self.NAME,
+            }
+        for key in extras:
+            df[key] = extras[key]
+        df.rename(columns={'value': 'load_MW'}, inplace=True)
+
+        # Drop the couple of times around DST transition that we don't handle correctly
```

Comment:
The two hours around DST transition in the fall and one hour in the spring transition.   PJM has a 26 hour day in the fall, and a 23 hour day in the spring.  I haven't looked into what is going on, or why this is different than the non-way-back formatting